app/controllers/api/v2/registration_controller.rb

```diff
@@ -92,7 +92,9 @@ def host
           return
         end
 
-        @host.setBuild
+        # Do not rebuild managed hosts
+        # https://bugzilla.redhat.com/show_bug.cgi?id=2212630
```

Comment:
Generally we avoid linking to BZs in code. https://projects.theforeman.org/issues/36393 already contains sufficient context so I'd prefer that if anything.